rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/tsdb/10_settings.yml

```diff
@@ -482,3 +523,36 @@ check end_time boundary with data_nano:
           "@timestamp": "2021-09-26T03:09:52.123456789Z",
           "metricset": "pod"
         }
+
+
+---
+check time_series default time bound value:
```

Comment:
I think we should add one more test with missing `start_time` and `end_time` bounds and make sure the default values are set. I have only seen such a test as a unit test.